dex/testing/firo/harness.sh

```diff
@@ -0,0 +1,331 @@
+#!/usr/bin/env bash
+SYMBOL="firo"
+FIRO_BIN_DIR="~/firo/bin"
+DAEMON="${FIRO_BIN_DIR}/firod"
+CLI="${FIRO_BIN_DIR}/firo-cli"
```

Comment:
These'll just need to be expected in `PATH`.

Reply:
Removed extra path variables in [119a661](https://github.com/decred/dcrdex/pull/2270/commits/119a661949de095ef6b7dc7ded2a1f2e55bbd585) below.